src/applications/mhv-medical-records/containers/RadiologyImagesList.jsx

```diff
@@ -41,7 +41,7 @@ const RadiologyImagesList = ({ isTesting }) => {
     isTesting || false,
   );
   const [isStudyJobsLoaded, setStudyJobsLoaded] = useState(isTesting || false);
-
+  const [diacomDownload, setDiacomDownload] = useState(false);
```

Comment:
The file is called is "dicom" as opposed to "diacom". Can you correct all the instances?